src/SprykerEco/Shared/Inxmail/InxmailConstants.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Inxmail;
+
+/**
+ * Declares global environment configuration keys. Do not use it for other class constants.
+ */
+interface InxmailConstants
+{
+    public const KEY_ID = 'INXMAIL:KEY_ID';
+    public const SECRET = 'INXMAIL:SECRET';
+
+    public const API_EVENT_URL = 'INXMAIL:API_EVENT_URL';
+
+    public const EVENT_CUSTOMER_REGISTRATION = 'INXMAIL:EVENT_CUSTOMER_REGISTRATION';
+    public const EVENT_CUSTOMER_RESET_PASSWORD = 'INXMAIL:EVENT_CUSTOMER_RESET_PASSWORD';
+    public const EVENT_ORDER_NEW = 'INXMAIL:EVENT_ORDER_NEW';
+    public const EVENT_ORDER_SHIPPING_CONFIRMATION = 'INXMAIL:EVENT_ORDER_SHIPPING_CONFIRMATION';
+    public const EVENT_ORDER_CANCELED = 'INXMAIL:EVENT_ORDER_CANCELED';
```

Comment:
Are you sure those are valid ENV configs as outlined in the class doc block above and per guidelines?
Also, cancelled has to L.

Reply:
L is updated. Event configuration is set on the Inxmail side and events can be different for env to env.